tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/domain/MetaSyntax.java

```diff
@@ -0,0 +1,52 @@
+package fi.helsinki.cs.tmc.langs.domain;
+
+public class MetaSyntax {
+    
+    private static final String BEGIN_SOLUTION = "BEGIN[ \\t]+SOLUTION";
+    private static final String END_SOLUTION = "END[ \\t]+SOLUTION";
+    private static final String SOLUTION_FILE = "SOLUTION[ \\t]+FILE";
+    private static final String SPACES = "[ \\t]*";
```

Comment:
Perhaps use this in the other constants as well, unless it ends up looking illegible.